docs/immutable/immutablejs.md

```diff
@@ -1,163 +0,0 @@
-# Using Immutable Data Structures
```

Comment:
I think that would be a good idea. The util is what we recommend, but some people might have reasons to do it manually.  Plus I think it does a good job as an example showing how you can hook into serialization etc.  Might give people ideas to integrate other immutable libs themselves.

tldr; never hurts to have more docs :smile: